src/grimoirelab/core/runner/commands/run.py

```diff
@@ -124,74 +118,42 @@ def eventizers(workers: int):
 @click.option('--workers',
               default=20,
               show_default=True,
-              help="Number of workers to run in the pool.")
-def archivists(workers: int):
-    """Start a pool of archivists workers.
+              help="Number of archivists to run.")
+@click.option("--verbose",
+              is_flag=True,
+              default=False,
+              help="Enable verbose mode.")
+@click.option("--burst",
+              is_flag=True,
+              default=False,
+              help="Process all the events and exit.")
+def archivists(workers: int, verbose: bool, burst: bool):
```

Comment:
When there's no opensearch server available, this runs forever trying to connect.